src/remote.js

```diff
@@ -23,7 +23,8 @@ function getTile(source, tile, done) {
 
   tileRequest(url, function(err, res, body) {
     if (err) return done(err);
+    else if (res.statusCode === 200) return done(null, parseVT(body, tile, source));
+    else if (res.statusCode === 404) return done(null, parseVT(null, tile, source));
```

Comment:
I think we should handle this the same as in MBTiles, not doing any parsing and passing `undefined` as the reduce value which makes the main thread not fire a `reduce` event. https://github.com/mapbox/tile-reduce/blob/nextgen/src/mbtiles.js#L33